documentation/api/metrics/v1/mbeans.markdown

```diff
@@ -111,6 +111,7 @@ following `<item>`s:
   previously enqueued commands
 * `queue-time`: histogram of the time commands have spent waiting in the queue
 * `depth`: number of currently enqueued commands
+* `bashed`: number of commands which have been "bashed" in the command queue
```

Comment:
I'm not sure, but while "bashed" is quite specific, i wondered whether something like "suppressed" might be preferable as a public name, ie.. 'number of "stale/obsolete/obviated/whatever' commands that have been suppressed".  On the other hand, maybe metrics are "internal" enough that the name more directly linked to the operation is better...

Reply:
That's a good point. I went with "ignored" because that's how we refer to it in the logs [here](https://github.com/puppetlabs/puppetdb/blob/master/src/puppetlabs/puppetdb/command.clj#L282)